src/integrations/mssql/sqlServerGenerator.ts

```diff
@@ -88,6 +88,8 @@ export default class SqlServerGenerator extends Generator {
             isNullArguments = `CONVERT(VARCHAR, ${column.escapedName}), 'NULL'`;
         } else if (this.includesDataType(column.objectExplorerLabel, ["date", "time"])) {
             isNullArguments = `'''' + CONVERT(VARCHAR, ${column.escapedName}) + '''', 'NULL'`;
+        } else if (this.includesDataType(column.unsanitizedAttributes, ["text"])) {
```

Comment:
This file changed just slightly so we'll just need to update a couple things.

```suggestion
        } else if (this.includesDataType(column.objectExplorerLabel, ["text"])) {
```

Reply:
good catch, I didn't even think to look for changes with the new setup.